INSTALL.md

```diff
@@ -49,6 +49,16 @@ If a package was installed after a failed compilation, use "make mrproper-top" t
 
 Additional packages maybe needed, please refer to: https://criu.org/Installation
 
+In order to allow multiple installations to work in conjunction with one another (i.e., no conflicts or shadow imports)
+the environment variable `PY_PKG_DIR` which defaults to "pycriu" (i.e., `import pycriu`).
```

Comment:
Here you mean that we must set `PY_PKG_DIR`?

Reply:
oh, something is truncated here, Let me amend it.